### PR TITLE
DNN-10337 Fixed wrong condition not allowing user deletion.

### PR DESCRIPTION
**Pull request by mean2me**
Added missing messages in referenced .resx file.

### CODE REVIEW

File: src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/Prompt/Commands/PurgeUser.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public override ConsoleResultModel Run()
             var userInfo = UserController.Instance.GetUser(PortalId, UserId);
             if (userInfo == null)
                 return new ConsoleErrorResultModel(string.Format(LocalizeString("UserNotFound"), UserId));
-            if (!userInfo.IsDeleted)
+            if (userInfo.IsDeleted)
```
Comment (kanm2015) [inline src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/Prompt/Commands/PurgeUser.cs]: only purge soft deleted user, should not change here

File: src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/RecyclebinController.cs [hunk 2]
```diff
@@ -332,7 +332,7 @@ public void DeleteUsers(IEnumerable<UserInfo> users)
                 var user in
                     userInfos.Select(u => UserController.GetUserById(u.PortalID, u.UserID))
                         .Where(user => user != null)
-                        .Where(user => user.IsDeleted))
+                        .Where(user => !user.IsDeleted))
```
Comment (kanm2015) [inline src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/RecyclebinController.cs]: only purge soft deleted user, should not change here

File: src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/Prompt/Commands/PurgeUser.cs (hunk 1 above)
Comment (mean2me) [reply src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/Prompt/Commands/PurgeUser.cs]: What's the correct logic? As it was before, only soft-deleted users will be purged. With this new logic, if a user has been softly deleted (for some reason) I don't purge it.

File: src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/RecyclebinController.cs (hunk 2 above)
Comment (mean2me) [reply src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/RecyclebinController.cs]: Same as previous comment.

**Comment by kanm2015** [comment]
code review done

**Pull request merged**